### PR TITLE
[Merged by Bors] - Add cubic curve section to v0.10 release blog post

### PR DESCRIPTION
**Pull request by aevyrie**

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
You'll want to rebase this to avoid the weird diff :)

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 1]
```diff
@@ -739,6 +584,17 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## Cubic Curves
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+In preparation for UI animation and animation editing, cubic curves have been added to `bevy_math` with a public `CubicGenerator` trait to enable user-defined cubic curves. The implementation provides multiple curves out of the box, useful in various applications:
```
Comment (alice-i-cecile) [inline content/news/2023-03-04-bevy-0.10/index.md:591]:
```suggestion
In preparation for UI animation and hand-tweaked animation curves cubic curves have been added to `bevy_math` with a public `CubicGenerator` trait to enable user-defined cubic curves. The implementation provides multiple curves out of the box, useful in various applications:
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 2]
```diff
@@ -739,6 +584,17 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## Cubic Curves
```
Comment (alice-i-cecile) [inline content/news/2023-03-04-bevy-0.10/index.md:587]: I'd like to see a note on their performance here. It's really impressive!

**Comment by aevyrie** [comment]
Damn you're quick, was just fixing the weird diff 😅

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 3]
```diff
@@ -739,6 +584,17 @@ query.iter().for_each(|mut component| {
...
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+In preparation for UI animation and animation editing, cubic curves have been added to `bevy_math` with a public `CubicGenerator` trait to enable user-defined cubic curves. The implementation provides multiple curves out of the box, useful in various applications:
+
+- `Bezier`: user-drawn splines, and cubic-bezier animation easing for UI - helper methods are provided for cubic animation easing.
+- `Hermite`: useful for smooth interpolation betwen two points in time where you know both the position and velocity, such as network prediction.
+- `Cardinal`: interpolates between any number of control points, automatically computing tangents; Catmull-Rom is a type of Cardinal spline.
+- `B-Spline`: the only C2 continuous curve, particularly useful for smooth camera motion where continuous acceleration is important.
```
Comment (alice-i-cecile) [inline content/news/2023-03-04-bevy-0.10/index.md:596]: C2 continuous is very jargony. Even discussing continuity / smoothness of derivatives would be more approachable.

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 4]
```diff
@@ -739,6 +584,17 @@ query.iter().for_each(|mut component| {
...
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## Cubic Curves
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+In preparation for UI animation and animation editing, cubic curves have been added to `bevy_math` with a public `CubicGenerator` trait to enable user-defined cubic curves. The implementation provides multiple curves out of the box, useful in various applications:
+
+- `Bezier`: user-drawn splines, and cubic-bezier animation easing for UI - helper methods are provided for cubic animation easing.
+- `Hermite`: useful for smooth interpolation betwen two points in time where you know both the position and velocity, such as network prediction.
```
Comment (LiamGallagher737) [inline content/news/2023-03-04-bevy-0.10/index.md:594]:
```suggestion
- `Hermite`: useful for smooth interpolation between two points in time where you know both the position and velocity, such as network prediction.
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 5]
```diff
@@ -739,6 +584,17 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## Cubic Curves
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+In preparation for UI animation and hand-tweaked animation curves cubic curves have been added to `bevy_math` with a public `CubicGenerator` trait to enable user-defined cubic curves. The implementation provides multiple curves out of the box, useful in various applications:
```
Comment (alice-i-cecile) [inline content/news/2023-03-04-bevy-0.10/index.md:591]:
```suggestion
In preparation for UI animation and hand-tweaked animation curves, cubic curves have been added to `bevy_math`.  The implementation provides multiple curves out of the box, useful in various applications:
```

Then below discuss the  "public `CubicGenerator` trait to enable user-defined cubic curves".

File: content/news/2023-03-04-bevy-0.10/index.md (hunk 3 above)
Comment (alice-i-cecile) [reply content/news/2023-03-04-bevy-0.10/index.md:596]: Linking to the wikipedia page would also be okay with me, if you prefer to keep the language.

**Review by alice-i-cecile — APPROVED** [review]

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 6]
```diff
@@ -739,6 +739,26 @@ query.iter().for_each(|mut component| {
...
+## Cubic Curves
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+<video controls loop><source  src="cubic_curves.mp4" type="video/mp4"/></video>
+<p class="release-feature-authors">This video shows four kinds of cubic curves being smoothly animated with bezier easing. The curve itself is white, green is velocity, red is acceleration, and blue are the control points that determine the shape of the curve.</p>
+
+In preparation for UI animation and hand-tweaked animation curves, cubic curves have been added to `bevy_math`.  The implementation provides multiple curves out of the box, useful in various applications:
+
+- `Bezier`: user-drawn splines, and cubic-bezier animation easing for UI - helper methods are provided for cubic animation easing as demonstrated in the above video.
```
Comment (alice-i-cecile) [inline content/news/2023-03-04-bevy-0.10/index.md:751]: List style is inconsistent here, so CI is failing.

**Review by LiamGallagher737 — APPROVED** [review]

File: content/news/2023-03-04-bevy-0.10/index.md (hunk 3 above)
Comment (aevyrie) [reply content/news/2023-03-04-bevy-0.10/index.md]: I ended up removing jargon.

**Review by cart — APPROVED** [review]

**Pull request closed without merging**